README.md

```diff
@@ -55,6 +55,9 @@ Requires the following to be installed on the deployment host:
 - [skopeo](https://github.com/containers/skopeo)
 - [podman](https://github.com/containers/podman/)
 - [kubectl + oc](https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html)
+- [pyghmi](https://pypi.org/project/pyghmi/) #For PXE deployment
+- [ipmitool](https://github.com/ipmitool/ipmitool) #For PXE deployment
```

Comment:
Can you add `python3-pyghmi` and `ipmitool` to line 67 the `dnf install`

Reply:
DONE